.travis.yml

```diff
@@ -14,73 +14,97 @@ matrix:
     - python: 2.7
       env: FIREFOX=nightly
       addons:
+        apt:
+          packages:
+            - jwm
         firefox: latest-nightly
     - python: 2.7
       env: FIREFOX=dev
       addons:
+        apt:
+          packages:
+            - jwm
         firefox: latest-dev
     - python: 2.7
       env: FIREFOX=beta
       addons:
+        apt:
+          packages:
+            - jwm
         firefox: latest-beta
     - python: 2.7
       env: FIREFOX=release
       addons:
+        apt:
+          packages:
+            - jwm
         firefox: latest
-    - python: 2.7
-      env: FIREFOX=esr
-      addons:
-        firefox: latest-esr
+    # - python: 2.7
```

Comment:
I suspect the comments might be part of the Travis CI failure. I'm going to remove these sections entirely, but I can add an issue to add ESR support once 52.1.0 ships.